resources/js/components/Committee.vue

```diff
@@ -0,0 +1,65 @@
+<script>
+import CommitteeInfo from './CommitteeInfo';
+import CommitteeMemberList from './CommitteeMemberList';
+import CommitteeSessions from './CommitteeSessions';
+export default {
+    name: 'Committee',
+    props: {
+        info: {
+            type: String,
+            default: () => 'info'
+        },
+        members: {
+            type: Array,
+            default: () => []
+        },
+        meetings: {
+            type: Array,
+            default: () => []
+        }
+    },
+    components: {
+        CommitteeInfo,
+        CommitteeMemberList,
+        CommitteeSessions
+    },
+    data() {
+        return {
+            activeTab: 'mitglieder'
+        };
+    },
+};
+</script>
+
+<template>
+    <div class="">
+        <nav class="ris-committee-navigation ris-without-padding-mob">
+            <ul class="ris-ul ris-committee-navigation__list">
+                <li class="ris-committee-navigation__item">
+                    <button
+                        @click="activeTab = 'aufgaben'"
+                        :class="['ris-committee-navigation__button ris-committee-navigation__button--aufgaben', activeTab === 'aufgaben' ? 'active' : '' ]">
+                        Aufgaben
+                    </button>
+                </li>
+                <li class="ris-committee-navigation__item">
```

Comment:
further improvement would be to display `.ris-committee-navigation__item` elements  in a loop since they are mostly the same and you can avoid duplicating the same piece of code 3 times; for example you can define the list of tabs in `data`
```
data: () => ({
   tabs: ['aufgaben', 'mitglieder', 'sitzungen']
})
```
and then loop through them and with `v-for`